### PR TITLE
Allow props override for Formik Select

### LINKED ISSUE #791: onChange not triggering in Select formik field.
Currently we don't have the ability to add onChange handler in Select formik field. In some of the apps we have a use-case where we need to store only the value of the options instead of option object. In this scenario it's better that we can provide our own onChange handler.

### PR DESCRIPTION
**Pull request by ajmaln**
- Fixes #791 

@ajmaln _a

### CODE REVIEW

**Comment by ajmaln** [comment]
@amaldinesh7 _a Please review.

File: lib/components/formik/Select.js [hunk 1]
```diff
@@ -43,6 +42,7 @@ const SelectField = ({
       error={meta.touched && meta.error}
       getOptionValue={getOptionValue || ((option) => option.value)}
       isMulti={!!isMulti}
+      {...rest}
```
Comment (amaldinesh7) [inline lib/components/formik/Select.js:45]: isn't `otherProps` a better name? or that's what we've followed throughout the project.

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@ajmaln _a

**Comment by ajmaln** [comment]
@amaldinesh7 _a updated the props name.

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**